terragrunt/aws/api/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "function_arn" {
+  value = module.url_shortener_lambda.function_arn
+}
+
+output "function_name" {
+  value = module.url_shortener_lambda.function_name
+}
+
+output "invoke_arn" {
+  value = module.url_shortener_lambda.invoke_arn
+}
```

Comment:
Could you add a `description` to each when you get a chance.

Reply:
Done! Now each output value has a description.